.bazelrc

```diff
@@ -8,6 +8,8 @@ startup --host_jvm_args=-Xmx512m
 build --workspace_status_command=bazel/get_workspace_status
 build --experimental_remap_main_repo
 build --host_force_python=PY2
+build --action_env=BAZEL_LINKLIBS=-l%:libstdc++.a
```

Comment:
What effect does this have on macOS or Windows builds? Is there any way to set it only on Linux?

Reply:
From my read of https://github.com/bazelbuild/bazel/tree/0.28.0/tools/cpp this env doesn't have any effect on `configure_osx_toolchain` or `configure_windows_toolchain` so it should be no-op.